src/deprecated.jl

```diff
@@ -30,3 +30,13 @@ function get_bids(system::SystemDA, type_of_bid::Symbol)
         return getproperty(system, type_of_bid)
     end
 end
+
+@deprecate get_regulation_offers(system::System) get_regulation_offers(get_ancillary_services(system))
```

Comment:
We might not want to bother deprecating.  I'm not sure it would work well given the name of the function has stayed the same but it acts on a different type in the new design.

Reply:
Deprecating based on types works just fine. I wouldn't worry too much about getters during an RFC PR though since things might change drastically